From 06b667de02df34e3efa1864b4031853bd4ee010d Mon Sep 17 00:00:00 2001
From: Yair Ben-Avraham <yairba@protonmail.com>
Date: Sun, 10 Jan 2021 06:34:58 +0100
Subject: [PATCH] castore: fix build without lzma or xz

castore.c unconditioanlly includes lzma.h, but lzma support if optional.
Furthermore, castore.c does not need anything from lzma.h.

Drop the include altogether.

Signed-off-by: Yair Ben-Avraham <yairba@protonmail.com>
[yann.morin.1998@free.fr:
  - lzma.h is not needed at all
  - rewrite commit log accordingly
]
Signed-off-by: Yann E. MORIN <yann.morin.1998@free.fr>
---
 src/castore.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/castore.c b/src/castore.c
index c290dfd..b2e5320 100644
--- a/src/castore.c
+++ b/src/castore.c
@@ -2,7 +2,6 @@
 
 #include <dirent.h>
 #include <fcntl.h>
-#include <lzma.h>
 #include <sys/stat.h>
 #include <unistd.h>
 
-- 
2.25.1

